src/ert/_c_wrappers/job_queue/driver.py

```diff
@@ -47,7 +47,7 @@ class Driver(BaseCClass):
     def __init__(
         self,
         driver_type: QueueDriverEnum,
-        max_running: int = 1,
+        max_running: int = 0,
```

Comment:
This has been `1` for 12 years (https://github.com/equinor/ert/commit/2d78a3004977f1679ce6c179124d0760b41211df). 

Has this default been inactive for all that time? Is there is a specific git commit that suddenly made this default active?

Reply:
Yes, it was "inactive" as you say. The Driver was created through the c code where it would get the value 0.